app/embeds/youtube.js

```diff
@@ -0,0 +1,32 @@
+// @flow
```

Comment:
A lot still in the air, but I'm super happy with this - embeds can be added to Outline by defining a single React component like this.

Configuration is defined as statics.